libs/core/core/components/apps/help-app.component.tsx

```diff
@@ -23,6 +24,7 @@ const getStoreData = async (key?: string) =>
 
 const HelpApp: FunctionComponent = () => {
   useAltLinkDownloadPreventer()
+  useBackForwardButtonPreventer()
```

Comment:
This file is going to be removed in https://github.com/mudita/mudita-center/pull/2033 so you can revert those changes to avoid conflicts.